doc/Extensions/Network-Map.md

```diff
@@ -12,55 +14,59 @@ $config['network_map_items'] = array('mac','xdp');
 ```
 
 Either remove mac or xdp depending on which you want.
+XDP is based on FDP, CDP and LLDP support based on the device type.
 
 A global map will be drawn from the information in the database, it is worth noting that this could lead to a large network map.
 Network maps for individual devices are available showing the relationship with other devices.
+Also you can Build Device Groups and those Device Groups can be drawn with Network Map.
```

Comment:
Can you?

Reply:
Yes -  Overview -> Maps -> Device Groups Maps

![image](https://user-images.githubusercontent.com/27959419/32074779-1a531e2a-ba60-11e7-925f-e0fc0b757fc2.png)

Reply:
I did not know that :)